lib/email/message_builder.rb

```diff
@@ -25,6 +25,18 @@ def initialize(to, opts = nil)
           "user_notifications.header_instructions",
           @template_args,
         )
+        reply_by_email_key =
+          DiscoursePluginRegistry.apply_modifier(
+            :message_builder_modifier,
```

Comment:
We need unique names for these modifier so they can be used individually.. maybe `messge_builder_reply_by_email_key` and `messge_builder_respond_link_key`